src/classes/HMSSDK.tsx

```diff
@@ -1288,6 +1290,47 @@ export class HMSSDK {
     return await HMSManager.getSessionMetaData({ id: this.id });
   };
 
+  getVideoTrackLayerDefinition = async (remoteVideoTrack: HMSRemoteVideoTrack) => {
```

Comment:
These functions are to be added in `HMSRemoteVideoTrack` class not on the `HMSSDK` instance

```kotlin
fun setLayer(HMSLayer: HMSLayer)
        
fun getLayerDefinition() : List<HMSSimulcastLayerDefinition>

fun getLayer() : HMSLayer
```